src/main/resources/application.yml

```diff
@@ -1,5 +1,5 @@
 server:
-  port: 8081
+  port: 8180
```

Comment:
It should be 8081 as in other spring applications, I think rancher pipelines use this default port for deploying